packages/zero-cache/src/config/zero-config.test.ts

```diff
@@ -359,11 +359,6 @@ test('zero-cache --help', () => {
                                                                                                                                                                    
      --target-client-row-count number                            default: 20000                                                                                    
        ZERO_TARGET_CLIENT_ROW_COUNT env                                                                                                                            
-                                                                 The target number of rows to keep per client in the client side cache.                            
```

Comment:
@darkgnotic Is it not supposed to show anything if deprecated? Maybe I misunderstood how to use the deprecated config option?

Reply:
Is it warning when using the deprecated option?